projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-searching/table-searching.component.html

```diff
@@ -21,11 +21,13 @@ <h4 class="go-heading-4">searchable</h4>
         </p>
       </div>
 
-      <div class="go-column go-column--100">
+      <div class="go-column go-column--100 go-column--no-padding">
         <h4 class="go-heading-4">searchTerm</h4>
         <p class="go-body-copy">
           This is the term that is being searched for. This is useful when implementing server side integration, as it is emitted
           as part of the <code class="code-block--inline">GoTableConfig</code> object and can be used for server side filtering.
+          <br/>
+          <caption>See <a [routerLink]="['../server-integration']">Server Integration</a> for more details.</caption>
```

Comment:
Should this have `class="go-link"` on it?

Reply:
Created a PR for this.